ferolino_jasmin/country.html

```diff
@@ -0,0 +1,14 @@
+<body>
+    <h1>Countries of the World</h1>
+    <input type="text" id="countryInput" placeholder="Enter a country">
+    <button onclick="searchCountry()">Search</button>
+
+    <div id="countryDetails">
+    </div>
+
+    <div id="countriesInRegion">
+    </div>
+
+    <script src="countries.js"></script>

```

Comment:
This is not the name of your js file. javascript.js is the appropriate file name.